codalab/lib/bundle_cli.py

```diff
@@ -674,7 +674,25 @@ def print_table(
         for row_dict in row_dicts:
             row = []
             for col in columns:
-                cell = row_dict.get(col)
+                # These fields will not be returned by the server if the
+                # authenticated user is not root
+                if col in ('last_login', 'email', 'time', 'disk', 'parallel_run_quota'):
```

Comment:
Not sure why this logic is needed.  We should ideally just print out whatever fields the server sends back.